js/05/main.js

```diff
@@ -0,0 +1,47 @@
+// First remove all the punctuations and change the string to array and count the number of words in the array
+let text = 'I love teaching and empowering people. I teach HTML, CSS, JS, React, Python.';
+const words = text.split(' ');
+console.log(words);
+console.log(words.length);
+
+//In countries array check if 'Ethiopia' exists in the array if it exists print 'ETHIOPIA'. If it does not exist add to the countries list and print the array.
+let ethiopia = 'Ethiopia';
```

Comment:
If you won't change the variable's value it is better to use const instead of let. Your code will be more stable in that case and will help you to avoid errors in future. Check all other cases in this task, please.